packages/ui/components/document/document-share-button.tsx

```diff
@@ -142,8 +142,8 @@ export const DocumentShareButton = ({
           <DialogTitle>Share your signing experience!</DialogTitle>
 
           <DialogDescription className="mt-4">
-            Don't worry, the document you signed or sent wont be shared; only your signing
-            experience is. Share your signing card and showcase your signature!
+            Don't worry, the document you signed or sent won't be shared; only your signing
```

Comment:
I'd change it altogether to something like this:

```
Rest assured, your document is strictly confidential and will never be shared. Only your signing experience will be highlighted. Share your personalized signing card to showcase your signature!
```

Reply:
Sounds great! Updated.